CONP_DATS_fields.md

```diff
@@ -1,47 +1,53 @@
-|dates|**OPTIONAL**.  Relevant dates for the dataset. If you provide a date, it must come with a description of the date.|<!--will later choose from a pulldown list--> 
-|citations|**OPTIONAL**.  Publication(s) citing this dataset.|
-|citationCount|**OPTIONAL**. The number of publications that cite this dataset (enumerated in the citations property).|
-|producedBy|**OPTIONAL**. Process which generated a given dataset.|
-|isAbout|**OPTIONAL**. Entities (biological entity, taxonomic information, disease, molecular entity, anatomical part, treatment) associated with this dataset.|
-|hasPart|**OPTIONAL**. A dataset that is a subset of this dataset; datasets declaring the 'hasPart' relationship are considered a collection of datasets.  The aggregation criteria should be included in the 'description' field.|
-|acknowledges|**OPTIONAL**. Grant(s) which funded and supported the work reported by the dataset.|
-|refinement|**OPTIONAL**. Qualifier describing the level of data processing of the dataset and its distributions.|
-|aggregation|**OPTIONAL**. Qualifier indicating whether the entity represents an 'instance of a dataset' or a 'collection of datasets'.|
-|spatialCoverage|**OPTIONAL**. The geographical extension and span covered by the dataset and its measured dimensions/variables.|
+|Portal field name|DATS schema field|Status|Description|Portal help text|
+|-|-|-|-|-|
+|Title|title|**REQUIRED**| The name of the dataset. |The name of the dataset. The title needs to be short and easily recognizable and searchable. If an acronym, please also provide the long name.|
+|Creators|creators|**REQUIRED**| The person(s) or organization(s) which contributed to the creation of the dataset. | The person(s) or organization(s) which contributed to the creation of the dataset. This can be the principal investigator, hospital, university, centre, clinic, etc. The first contact provided will be associated as the main contact for this dataset.|
+|Description|description|**REQUIRED**| A textual narrative comprised of one or more statements describing the dataset.| A short paragraph providing a rapid overview of the dataset and the context of data collection. Suggestion of items to include in the description (if applicable): main use of the dataset, population studied, study design, sample size, data collected, methods, techniques, apparatus used to generate the data.|
+|Data types|types|**REQUIRED**| Terms to describe the nature of the data. | Terms to describe the nature of the data. Data type can be single or multiple. Add a term with the [interlex URI](https://neuinfo.org/interlex/dashboard) if possible.| 
+|Version|version|**REQUIRED**| Provide the version number, or the release point of your dataset.|Provide the version number, or the release point of your dataset. Need to be a positive number. |
```

Comment:
@PapillonMcGill maybe give some example of version here?

Reply:
Also, that should be  "Need**s** to be a positive number"

Reply:
Done.